queries/futures/useGetFuturesOrders.ts

```diff
@@ -0,0 +1,77 @@
+import { NetworkId } from '@synthetixio/contracts-interface';
+import Wei from '@synthetixio/wei';
+import { utils as ethersUtils } from 'ethers';
+import request, { gql } from 'graphql-request';
+import { useQuery, UseQueryOptions } from 'react-query';
+import { useRecoilValue } from 'recoil';
+
+import { ETH_UNIT } from 'constants/network';
+import QUERY_KEYS from 'constants/queryKeys';
+import Connector from 'containers/Connector';
+import useIsL2 from 'hooks/useIsL2';
+import { futuresAccountState, marketInfoState } from 'store/futures';
+import logError from 'utils/logError';
+
+import { getFuturesEndpoint } from './utils';
+
+const useGetFuturesOpenOrders = (options?: UseQueryOptions<any>) => {
```

Comment:
I noticed this is a separate query to `useGetFuturesOpenOrders` so worth renaming to `useGetFuturesOrders`. But also is there a reason we have two very similar queries now in those two? Can we combine them and query by passing accountType to differentiate between cross margin and isolated margin, similar to here: https://github.com/Kwenta/kwenta/blob/dev/queries/futures/useGetFuturesTradesForAccount.ts

Reply:
One or the other will be removed. We don't need to pass accountType because I'm using `selectedFuturesAddress` which takes the type into account.

Reply:
Might have been my misunderstanding of abstractAccount, so this will either be wallet address for isolated margin trades and cross margin address for cross margin trades?

Reply:
Exactly, which makes this so we can just check if `selectedFuturesAddress == abstractAccount`. If cross margin, they are both the smart contract. If isolate margin, they are both the EOA address.

This leaves open the option for us to query where `account` is equal to the EOA address, and we will get open orders for isolated _and_ cross margin.

Reply:
Ah perfect, I can update the other queries as well then instead of using accountType. I initially thought abstract account was only assigned on CM trades.